.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,52 @@
+<!-- PLEASE REMOVE THIS COMMENT BLOCK BEFORE SUBMITTING
+
+Thanks for filing an issue!  Before hitting the button, please read this.
+
+If this is a REQUEST FOR HELP, it is probably be better served by:
+  - Stack Overflow: http://stackoverflow.com/questions/tagged/kubernetes
+  - Mailing list: https://groups.google.com/forum/#!forum/google-containers
+  - Slack: http://slack.kubernetes.io/
+
+Please check for duplicates.  We know searching for dups can be tricky, but we
+appreciate that you try.
+  - If you find an issue that sounds like your bug/feature, reply there.
+  - If you just want to say "me too", PLEASE DO NOT REPLY. Instead, use the
+    smiley-face icon in the upper-right of the issue message.
+
+If this is a BUG REPORT, please:
+  - Fill in as much of the template below as you can.  If you leave out
+    information, we can't help you as well.
+
+If this is a FEATURE REQUEST, please:
+  - Describe *in detail* the feature/behavior/change you'd like to see.
+
+In both cases, be ready for followup questions, and please respond in a timely
+manner.  If we can't reproduce a bug or think a feature already exists, we
+might close your issue.  If we're wrong, PLEASE feel free to reopen it and
+explain why.
+-->
+
+----------
+BUG REPORT
+----------
+
+*Kubernetes version*: (try `kubctl version`)
+
+
+*Environment (e.g. cloud provider, OS, kernel, install tools)*:
+
+
+*What happened*:
+
+
+*What you expected to happen*:
+
+
+*How to reproduce it, as minimally and precisely as possible*:
+
+
+*Anything else do we need to know*:
+
+
+
+[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/.github/ISSUE_TEMPLATE.md?pixel)]()
```

Comment:
The [documentation](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) is vague here but I believe we need a .md extension if we want the templates to interpret markdown.  And with a .md extension comes the need for it to pass mungedocs.